daml-assistant/daml-sdk/src/main/scala/com/digitalasset/daml/sdk/SdkMain.scala

```diff
@@ -22,6 +23,7 @@ object SdkMain {
       case "trigger" => Trigger.main(rest)
       case "script" => Script.main(rest)
       case "test-script" => TestScript.main(rest)
+      case "export" => Export.main(rest)
```

Comment:
I've implemented the change, the command is now exposed as `daml ledger export script`. It's an entirely different way of exposing the command. So, I decided to do it right away instead of a follow-up PR, as that follow-up would essentially undo this one.

I've replicated the command-line argument parsing in optparse-applicative. The duplication is unfortunate, however, this way all arguments have user visible help strings. Otherwise, I didn't find a way to access the help of the underlying Scala option parser, even with `daml ledger export -- --help` optparse-applicative swallowed the `--help` instead of passing it through to the Scala side.